src/staticwebapp/azext_staticwebapp/_clients.py

```diff
@@ -0,0 +1,138 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+
+from azure.cli.core.util import send_raw_request
+from azure.cli.core.commands.client_factory import get_subscription_id
+
+
+API_VERSION = "2022-03-01"
+
+
+class DbConnectionClient():
+    @classmethod
+    def create_or_update(cls, cmd, resource_group_name, name, environment, connection_name="default",
+                         db_connection=None):
+        management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
+        api_version = API_VERSION
+        sub_id = get_subscription_id(cmd.cli_ctx)
+        if environment:
+            url_fmt = ("{}/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Web/staticsites/{}/builds/{}"
+                       "/databaseConnections/{}?api-version={}")
+            request_url = url_fmt.format(
+                management_hostname.strip('/'),
+                sub_id,
+                resource_group_name,
+                name,
+                environment,
+                connection_name,
+                api_version)
```

Comment:
May I ask why dose the `DbConnectionClient` request REST service directly? Could you consider using Code Gen V2 to implement this new extension? + Code Gen owner @kairu-ms be aware.

Reply:
Since this is a new feature, the REST API specs were subject to change as I was writing these commands. Is it okay if we this release as-is and circle back to reimplement these commands with code gen V2 later?

Reply:
OK, sounds good~